plugins/source/aws/docs/tables/aws_ram_principals.md

```diff
@@ -14,8 +14,9 @@ The composite primary key for this table is (**account_id**, **id**).
 |_cq_parent_id|UUID|
 |account_id (PK)|String|
 |region|String|
-|id (PK)|String|
 |creation_time|Timestamp|
 |external|Bool|
+|id (PK)|String|
 |last_updated_time|Timestamp|
-|resource_share_arn|String|
\ No newline at end of file
+|resource_share_arn|String|
```

Comment:
Could also add `resource_share_arn` as a PK... ResourceOwner is either `SELF` or `OTHER-ACCOUNTS` not sure if that would be enough

Reply:
I'd rather add `resource_share_arn` to the PK, too